cli/command/context/export-import_test.go

```diff
@@ -0,0 +1,99 @@
+package context
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/docker/cli/cli/config"
+	"github.com/docker/cli/cli/config/configfile"
+	"gotest.tools/assert"
+)
+
+func TestExportImportWithFile(t *testing.T) {
+	contextDir, err := ioutil.TempDir("", t.Name()+"context")
+	assert.NilError(t, err)
+	defer os.RemoveAll(contextDir)
+	contextFile := filepath.Join(contextDir, "exported")
+	cli, cleanup, err := makeCliMock(t)
+	assert.NilError(t, err)
+	defer cleanup()
+	createTestContextWithKube(t, cli)
+	assert.NilError(t, runExport(cli, &exportOptions{
+		contextName: "test",
+		dest:        contextFile,
+	}))
+	assert.NilError(t, runImport(cli, importOptions{
```

Comment:
You check here that import does not fail... But you don't check the result. It may succeed but without importing anything. Same for the export, maybe you can golden the export?

Reply:
No need for a golden: I deepEqual the exported and imported context (in following commit)